components/automate-chef-io/content/docs/on-prem-builder.md

```diff
@@ -0,0 +1,66 @@
++++
+title = "Deploying Builder with Chef Automate"
+description = "use Chef Automate to deploy Builder"
+date = 2019-11-19T14:10:15-08:00
+weight = 20
+draft = false
+bref = ""
+toc = true
++++
+
+## Overview
+Enterprise customers may wish to set up an on-premises Habitat Builder depot to store
+Habitat packages for use by their own Habitat Studios and Supervisors. Use Chef Automate
+to deploy and manage an on-premises Habitat Builder depot.
+
+{{< warning >}}
+The current implementation of Automate-deployed Habitat Builder is an early milestone
+intended for use in proof-of-concept deployments. We do not consider it a fully-mature
+feature, and you should not either.
+{{< /warning >}}
+
+### System requirements
+<!-- Question: What do we tell them about disk space requirements for Builder artifact
+storage? -->
+
+Currently, Habitat Builder requires Chef Automate to be installed on the same host for
+authentication purposes. Outside the boundaries of a proof-of-concept exercise, we
+recommend against running production installations of Chef Automate and Habitat Builder on
+the same host.
+
+### Deploy on-premises Habitat Builder
+To deploy Chef Automate and Habitat Builder, run the command
+
+```shell
+chef-automate deploy --product builder --product automate
+```
+
+<!--
+Questions:
+1. If someone has an existing Chef Automate installation with the Builder bits, would it
+   be safe for them to deploy the builder product in that installation?
+2. How is licensing going to work?
+-->
+
+### Login to Habitat Builder
+1. Login to Chef Automate using the <relref "install.md#open-chef-automate">credentials provided during deployment.</relfef>
+1. Go to https://automate.example.com/bldr.
+1. Click the Sign In button in the upper right corner.
+1. Click Sign In With Chef Automate.
+1. Use the same credentials as in Step 1 to sign in to Habitat Builder.
+
+See the <ref "https://www.habitat.sh/docs/using-builder/">Habitat Builder documentation</ref> for setting up a Builder access token, creating a Builder origin, etc.
+
+The Automate-deployed Habitat Builder currently supports authentication with local users only.
+
+### Backup Habitat Builder
+<!-- Questions:
+  1. What is our commitment to forward compatibility of the backup format?
+  2. Do we intend to support Builder-only backups? Right now, if someone wants to backup
```

Comment:
Yeah, in practice those should be super small, but dealing with ElasticSearch for builder only deployment is :hankey: I'm firmly on the let's get it running without a hard-requirement-of-the-automate-product train.